doc/Terraform Example/jit.tf

```diff
@@ -0,0 +1,255 @@
+  address      = "34.xxx.xxx.xxx"
+}
+
+# use to route requests to back end services, can have rules
+resource "google_compute_url_map" "lb_default" {
+  project         = module.jit_project.project_id
+  name            = "jitaccess"
+  default_service = google_compute_backend_service.jit-backend.id
+}
+
+# google managed ssl certificates, to check the status run the following
+# gcloud compute ssl-certificates describe jitaccess \
+#     --global \
+#     --format="get(name,managed.status)"
+resource "google_compute_managed_ssl_certificate" "lb_default" {
+  provider = google-beta
+  project  = module.jit_project.project_id
+  name     = "jitaccess"
+
+  managed {
+    domains = ["jitaccess.corp.com"] #this needs to be the url that you configured as your DNS A record
+  }
+}
+
+# maps incomming https requests to a url map.
+resource "google_compute_target_https_proxy" "lb_default" {
+  provider = google-beta
+  project = module.jit_project.project_id
+  name    = "jitaccess-https-proxy"
+  url_map = google_compute_url_map.lb_default.id
+  ssl_certificates = [
+    google_compute_managed_ssl_certificate.lb_default.name
+  ]
+  depends_on = [
+    google_compute_managed_ssl_certificate.lb_default
+  ]
+}
+
+
+# forwards packets matching the ip and port range to a pool of vms.
+resource "google_compute_global_forwarding_rule" "google_compute_forwarding_rule" {
+  project               = module.jit_project.project_id
+  name                  = "jitaccess-https"
+  provider              = google-beta
+  load_balancing_scheme = "EXTERNAL"
+  port_range            = "443"
+  target                = google_compute_target_https_proxy.lb_default.id
+  ip_address            = google_compute_global_address.jitaccess-ip.id
+  depends_on = [
+    google_tags_tag_binding.binding
+  ]
+}
+
+########################################################################################################################
+#
+# Tag to allow access through the org policy
+#
+########################################################################################################################
+
+# Tag group
+resource "google_tags_tag_key" "key" {
+  parent      = "organizations/${local.org_id}"
+  short_name  = "jit-tag"
+  description = "Allow JIT an external load balancer."
+}
+
+# tag instance
+resource "google_tags_tag_value" "value" {
+  parent      = "tagKeys/${google_tags_tag_key.key.name}"
+  short_name  = "allowed"
+  description = "jit LB allow tag."
+}
+
+# bind tag to this project - note this is also harcoded into the org-policy
+resource "google_tags_tag_binding" "binding" {
+  depends_on = [
+    module.jit_project
+  ]
+  parent    = "//cloudresourcemanager.googleapis.com/projects/${module.jit_project.project_number}"
+  tag_value = "tagValues/${google_tags_tag_value.value.name}"
+}
+
+# Prevent external loadbalancer and only allow this one specific LB to be external
+resource "google_org_policy_policy" "restrict_lb_creation_by_lb_types" {
```

Comment:
@duxbuse - why did you apply this org policy? What affect would this have on the rest of `lbs` and external access in the org?